config/profile/rhdh/plugin-infra/orchestrator/infra-serverless.yaml

```diff
@@ -31,4 +31,4 @@ apiVersion: v1
 kind: Namespace
```

Comment:
When I tried to install openshift serverless operator standalone, from dev console (it was v 1.35.0 I think), I saw it created knative-eventing and knative-serving namespaces, curious if we need to create it ourselves?

Reply:
Let's remove them so?